### PR TITLE
[OSDEV-1146] Facility Claim Moderation Page. Claim Decision column is not downloaded.

### PR DESCRIPTION
**Pull request by roman-stolar**
[OSDEV-1146](https://opensupplyhub.atlassian.net/browse/OSDEV-1146) **Facility Claim Moderation Page. Claim Decision column is not downloaded.**

- Fixed issue with missed header & data for Claim Decision column

[OSDEV-1146]: https://opensupplyhub.atlassian.net/browse/OSDEV-1146?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Innavin369 — DISMISSED** [review]
LGTM

File: src/react/src/util/util.facilityClaimsXLSX.js [hunk 1]
```diff
@@ -18,8 +19,11 @@ const formatFacilityClaimsDataForXLSX = facilityClaims =>
             facilityClaim.contributor_name,
             facilityClaim.facility_country_name,
             moment(facilityClaim.created_at).format('LL'),
-            moment(facilityClaim.updated_at).format('LL'),
+            facilityClaim.claim_decision !== null
+                ? moment(facilityClaim.claim_decision).format('LL')
+                : 'N/A',
```
Comment (VadimKovalenkoSNF) [inline src/react/src/util/util.facilityClaimsXLSX.js:24]: Maybe it is better to replace this 'N/A' into a const and import it? We use it in `src/react/src/components/DashboardClaimsListTable.jsx` as well. What if client decide to change it to 'None' or simple '-' later?

**Review by VadimKovalenkoSNF — CHANGES_REQUESTED** [review]
Asked for minor change.

File: src/react/src/util/util.facilityClaimsXLSX.js (hunk 1 above)
Comment (roman-stolar) [reply src/react/src/util/util.facilityClaimsXLSX.js]: Fixed.

**Review by vladsha-dev — APPROVED** [review]
LGTM

**Review by mazursasha1990 — APPROVED** [review]
LGTM

**Pull request merged**